Kuroba/core-model/src/main/java/com/github/k1rakishou/model/data/post/ChanPost.kt

```diff
@@ -94,13 +94,9 @@ open class ChanPost(
 
   @Synchronized
   open fun allLoadersCompletedLoading(): Boolean {
-    for (loaderContentLoadState in onDemandContentLoadedMap.values) {
-      if (loaderContentLoadState.everythingLoaded()) {
-        return false
-      }
+    return onDemandContentLoadedMap.values.none {
+      it.everythingLoaded()
```

Comment:
I think there was a bug in the original code and it should be:

```kotlin
return onDemandContentLoadedMap.values.all { loaderContentLoadState ->
      loaderContentLoadState.everythingLoaded()
}
```

Reply:
👍